consensus_pbft.go

```diff
@@ -581,9 +589,11 @@ func (p *Pbft) runRoundChangeState(ctx context.Context) {
 		}
 	}
 
+	i := 1
 	// create a timer for the round change
 	for p.getState() == RoundChangeState {
-		_, span := p.tracer.Start(ctx, "RoundChangeState")
+		_, span := p.tracer.Start(ctx, fmt.Sprintf("RoundChange_Read#%d", i))
```

Comment:
This variable gets called `span` and `loopSpan`. I would prefer to call it `span` always since there are no other spans around.